src/status_im2/contexts/quo_preview/profile/profile_card/view.cljs

```diff
@@ -0,0 +1,108 @@
+           profile-picture?
+           name
+           hash
+           emoji-hash
+           on-press-dots?
+           on-press-sign?] :or {show-sign-profile? false
+                                key-card? false}}]
+  [rn/view {:style style/card-container}
+   [rn/view {:style style/card-header}
+    [user-avatar/user-avatar
+     {:full-name         name
+      :profile-picture   profile-picture?
+      :size              :medium
+      :status-indicator? false
+      :ring?             true}] ;; TODO(@mattgle): The ring? is not showing up for some reason
+    [quo2/button
+     {:size           24 ;; TODO(@mattgle): For some reason when I change the value to 32 the icon disappears
+      :type           :blur-bg
+      :icon           true
+      :on-press       on-press-dots?}
+     :i/more]]
+   [rn/view
+    {:style style/name-container}
+    [quo2/text
+     {:size   :heading-2
+      :weight :semi-bold
+      :number-of-lines 1
+      :style style/user-name} name]
+    (when key-card? (quo2/icon :i/key style/key-card))] ;; TODO(@mattgle): Change this for key-card icon
+   [quo2/text
+    {:weight   :monospace
+     :number-of-lines 1
+     :style style/user-hash} hash]
+   (when emoji-hash [quo2/text
+                     {:weight   :monospace
+                      :number-of-lines 1} emoji-hash]) ;; TODO(@mattgle): Ask about the emoji system that was mention in the issue
+   (when show-sign-profile? [button/button
+                             {:on-press            on-press-sign?
+                              :type :community
+                              :community-color  (colors/alpha "#397788" 0.6) ;; TODO(@mattgle): I don't know if this is the correct way
+                              :community-text-color colors/white
+                              :style style/sign-button} "Sign in to this profile"])]) ;; TODO(@mattgle): Should I use i18n here?
+
+
+;; cool-preview and preview-profile-card styles will not be extracted since they are just temporary
+
+(def descriptor
+  [{:label "Show: Sign in to this profile?"
+    :key   :show-sign-profile?
+    :type  :boolean}
+   {:label "Show: Is from key card?"
+    :key   :key-card?
+    :type  :boolean}
+   {:label "Hash"
+    :key   :hash
+    :type  :text}
+   {:label "Name"
+    :key   :name
+    :type  :text}
+   {:label "Emoji hash?"
+    :key   :emoji-hash
+    :type  :text}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:show-sign-profile?         true
+                             :key-card?                 true
+                             :name                      "Matt Grote"
+                             :on-press-dots?            nil
+                             :on-press-sign?            nil
+                             :profile-picture?           "https://elcomercio.pe/resizer/H_l7YL3q0NpRZ2Gh366nQ12L0NQ=/1200x900/smart/filters:format(jpeg):quality(75)/cloudfront-us-east-1.images.arcpublishing.com/elcomercio/PGCAFUVB3RCVZBKWDIVFRHLVO4.jpg"
```

Comment:
we shouldn't base64 image should be used, i believe we have exaples in the quo2 preview

Reply:
I tried converting that image to base64 and then asign that value to a variable but for some reasons that doesn't work. I've search for the components in `quo2 preview`, and the ones that has images are local resources and the component that I'm trying to use, uses the `uri:` in the `rn/image` so I can't use a local source. If I want to I would have to change the component and I don't think that's good since I don't find the use of having a diff prop for local images since it's an user avatar. In the `quo2 preview` the component `user-avatar` has the profile picture dropdown but if you choose a value breaks, because of this that I'm saying, they are treated as URLs and they are local resources. What should I do?
cc: @flexsurfer @ilmotta

Reply:
![image](https://user-images.githubusercontent.com/44204622/209854621-90d4ca34-f217-45dc-a744-521e0ad3a7cd.png)
This is for the base64 problem that I'm saying, for some reason (maybe cuz of the length) it's not treating it as a string

Reply:
@mattgle, data URIs need more information to work and from your screenshot it seems some bits are missing. You can find a working example in the code in the namespace `status-im2.contexts.quo-preview.community.data`.

Reply:
Thank you! Now it's working!

Reply:
Awesome!